src/view/com/util/fab/FABInner.tsx

```diff
@@ -42,14 +42,14 @@ export function FABInner({testID, icon, onPress, ...props}: FABProps) {
       ]}>
       <PressableScale
         testID={testID}
+        onPressIn={() => playHaptic('Light')}
         onPress={e => {
           playHaptic('Light')
-          setTimeout(
-            () => {
-              onPress?.(e)
-            },
-            isHapticsDisabled ? 0 : 75,
-          )
+          setTimeout(() => onPress?.(e), isHapticsDisabled ? 0 : 75)
```

Comment:
Just as a note, we probably want to get rid of this `setTimeout` here. I believe it was originally used for the composer FAB, in which case the timeout lets you see the animation of the fab before the sheet begins to present. In general though, we want to prefer fast response rather than seeing the animation imo